### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by Shoeman8** [community/community#67761] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How do I download Elastos Lite desktop wallet?

**Comment by Abhiram-J** [comment]
Try going through this webpage: https://appimage.github.io/elastos-light-wallet/

**Comment by Shoeman8** [comment]
Hi Abhiram, I don't have Linux system. Only a desktop Windows 10 that I need to use with my Ledger X to update Elastos lite wallet. Can you help?
Thanks!

Jim Shoemaker

Sent from Proton Mail mobile

-------- Original Message --------

**Comment by Abhiram-J** [comment]
HI Shoeman8,
I think the server is down for some reason. I guess this is the site from which you can download : https://elastos.info/blog/ela-wallets-guide/
But currently its down for me.
I found a reference site for the same [here](https://lotak612.medium.com/how-to-manage-elastos-on-ledger-with-the-elastos-light-wallet-51c69712c1aa)

**Comment by Shoeman8** [comment]
Hi Abhiram,
I receive a 404 error when trying to access Elastos desktop Windows download. I will continue to monitor it. My Elastos wallet has the correct address,

EHgdKcMxhNwzw3GWPrK1VDmxb37xZRQA2g

, but I cannot see the balance. When searching my address on Elastos blockexplorer, it gives no results, which has ne worried.

Thanks for your help

Jim Shoemaker

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
